buildconfig/CMake/CppCheck_Suppressions.txt.in

```diff
@@ -208,8 +208,8 @@ uselessCallsSubstr:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadFITS.cpp:5
 mismatchingContainerExpression:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadInstrument.cpp:148
 constVariableReference:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadIsawDetCal.cpp:277
 constVariableReference:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadIsawDetCal.cpp:286
+nullPointerOutOfResources:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadLog.cpp:445
 nullPointerOutOfResources:${CMAKE_SOURCE_DIR}/Framework/DataHandling/src/LoadLog.cpp:446
```

Comment:
Do we need both line 445 and 446, or just 445?

Reply:
So in the updated cppcheck suppression file it was lines 446 and 447 that were supressed. The changes you made deleted a line of code so I just moved them up. I'm presuming they need to be there.....

Reply:
Makes sense!